util/sys/storage/sys_other.go

```diff
@@ -0,0 +1,20 @@
+// Copyright 2020 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// +build !linux,!windows,!darwin
+
+package storage
+
+// GetTargetDirectoryCapacity get the capacity (bytes) of directory
+func GetTargetDirectoryCapacity(path string) (uint64, bool, error) {
+	return 0, false, nil
```

Comment:
```suggestion
func GetTargetDirectoryCapacity(path string) (uint64, error) {
	return max.Uint64, false
```

So that the return value of type `bool` is not necessary.